src/views/list.ts

```diff
@@ -0,0 +1,87 @@
+/*!
+ * © 2019 Atypon Systems LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  JATS_HTML_LIST_STYLE_MAPPING,
+  JatsStyleType,
+  ManuscriptNode,
+} from '@manuscripts/transform'
+
+import { getActualAttrs } from '../lib/track-changes-utils'
+import { BaseNodeProps } from './base_node_view'
+import BlockView from './block_view'
+import { createNodeOrElementView } from './creators'
+export class ListView<
+  PropsType extends BaseNodeProps
+> extends BlockView<PropsType> {
```

Comment:
I'm guessing the main purpose of this view is to handle Track Changes. If so I think we keep `dom` as the default (div) and build the ul/ol ourselves and copy the innerHTML from `contentDOM` into our ul/ol.